Android/app/src/main/java/com/example/kahye/common/models/Class.java

```diff
@@ -0,0 +1,121 @@
+package com.example.kahye.common.models;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Class implements Parcelable {
+    @SerializedName("classID")
+    @Expose
+    private Integer classID;
+    @SerializedName("className")
+    @Expose
+    private String className;
+    @SerializedName("expertName")
+    @Expose
+    private String expertName;
+    @SerializedName("minGuestCount")
+    @Expose
+    private Integer minGuestCount;
+    @SerializedName("maxGuestCount")
+    @Expose
+    private Integer maxGuestCount;
+    @SerializedName("availableTimeTable")
+    @Expose
+    private List<timeTable> availableTimeTable = new ArrayList<timeTable>();
+
+    public Integer getClassID() {
+        return classID;
+    }
+
+    public void setClassID(Integer classID) {
+        this.classID = classID;
+    }
+
+    public String getClassName() {
+        return className;
+    }
+
+    public void setClassName(String className) {
+        this.className = className;
+    }
+
+    public String getExpertName() {
+        return expertName;
+    }
+
+    public void setExpertName(String expertName) {
+        this.expertName = expertName;
+    }
+
+    public Integer getminGuestCount() {
+        return minGuestCount;
+    }
+
+    public void setminGuestCount(Integer minGuestCount) {
+        this.minGuestCount = minGuestCount;
+    }
+
+    public Integer getmaxGuestCount() {
```

Comment:
getMaxGuestCount (camel casing requires max to be capitalized).
Same for: 

setmaxGuestCount
getminGuestCount
setminGuestCount